### DISCUSSION TITLE
Stupid idea but why not add variables

### DISCUSSION THREAD

**Discussion by Lioncub901** [keiwando/toonsquid#178] [Ideas]
I was thinking of the idea of variable.

The idea came when I saw your path animation tutorial and I saw the circle spin I thought that could be used as a timer but I said that might not be efficient since we will have to manually change the text but I said if we had variable that would not be the case.

Symbol:
I also thought suppose we want to change the color of one of the symbols but not all we could change the variable defined in the symbol like component input (unity). I thought that we could used the variable as a slide to change the r, g, and b. Maybe we could used them to change the opacity of an input like opacity, x, y, width, height, corner radius, stroke width, and many other properties. I was thinking you could allow us to get value of properties like to get width.value to get width or cornerRadius.value to get the corner radius value. Maybe add a random function to produce a random number maybe to change the size of symbols randomly or add a random amount of symbols. I think if you add variables it was make symbols more customizable. I know this is too ambitious, but I just wanted to tell my ideas.

**Comment by Lioncub901** [comment]
I thought having something like this could help make a timer.

**Comment by kavlor** [comment]
Do you mean like sharing the attribute so when you change the attribute of one thing the other changes to the same?

**Reply by Lioncub901** [reply]
no like a variable. So I could have a variable which is a number called "timer" and I could use keyframing to change the values. Also was thinking that we could input variable to like to be able to change the color of a symbol through the variable. It was just an idea.
